ext/storage_sources/gcp_store/gcp_connection.h

```diff
@@ -36,7 +36,7 @@
 class gcp_connection {
     public:
     gcp_connection(const std::string &bucket_name);
-    int list_objects(std::vector<std::string> &objects) const;
```

Comment:
Let's also deal with prefix searching from the tech design. Basically meaning that if given a prefix string we only return back objects with the prefix. Also we don't need to assign a default value to list_single

Reply:
yep, it's added here: [600b09f](https://github.com/wiredtiger/wiredtiger/pull/8717/commits/600b09f6e6320c6b3a271bf48c7bbe49933a1788)